src/SoundList.js

```diff
@@ -9,7 +9,35 @@ function timeConvert(minutes) {
   return `${(min < 10 ? "0" : "") + min}:${secs < 10 ? "0" : ""}${secs}`;
 }
 
-const SoundList = ({ tracks, header, onSoundClick, className = "" }) => {
+// Psuedo code:
+// function highlightSound(track, sound) {
+//   let packSoundArr = pack sounds array;
+//   let similarSoundArr = similar array;
```

Comment:
Similarly, the `<SoundList />` component doesn't know the Similar sounds list: it just receives a `tracks` prop from a parent component.